server/src/models/activity.js

```diff
@@ -12,6 +12,42 @@ const genNotifications = (activityModel, transaction) => {
 		},
 		raw: true,
 		transaction: transaction,
+	}).then(usersToNotify => {
+		if (activityModel.type == 4 || activityModel.type == 5) { // If the activity was related to a WR or PB, we need to check for map notifications
```

Comment:
It's best to use the enum directly, and also to early return than to shift all the code to the right:
```suggestion
		if (activityModel.type !== ACTIVITY_TYPE.PB_ACHIEVED && activityModel.type !== ACTIVITY_TYPE.WR_ACHIEVED) {
				return usersToNotify;
		}
```